README.md

```diff
@@ -124,6 +124,7 @@ v.created_at                # When the `event` occurred
 v.whodunnit                 # If the update was via a controller and the
                             # controller has a current_user method, returns the
                             # id of the current user as a string.
+v.user                      # Gets the user model instance if any (works with device).                   
```

Comment:
Let's name this method `whodunnit_record`. Not everyone's app has "users". Also, I'd like it to be obvious that the id comes from the `whodunnit` column.

Also, what does "(works with device)" mean?